src/components/FarmLPCard/FarmLPCard.tsx

```diff
@@ -78,19 +77,14 @@ const FarmLPCard: React.FC<{
     stakedAmounts?.totalStakedBase,
   );
 
-  const poolRate = getRewardRate(stakingInfo.totalRewardRate);
-
-  const earnedUSD =
-    Number(stakingInfo.earnedAmount.toSignificant()) *
-    dQuicktoQuick *
-    quickPriceUSD;
+  const poolRate = getRewardRate(
+    stakingInfo.totalRewardRate,
+    returnTokenFromKey('DQUICK'),
```

Comment:
This should be resolved dynamically as well we should not make assumptions about the underlying token.

Reply:
Added `rewardToken` property to LP farms